migrations/20180603182250_create-users-table.js

```diff
@@ -1,7 +1,7 @@
 exports.up = function(knex, Promise) {
 	return knex.schema.createTable('users', (table) => {
 		table.increments();
-		table.string('pseudo').notNullable().unique().index();
+		table.string('pseudo').notNullable().unique().index(); // "index" is not useful here since we already put unique
```

Comment:
Then why is the line exactly like it was before ?

Reply:
I did not change it because it is a migration file. I corrected the problem into another migration.
This comment is here to show that this line was wrong and should not be taken into account